app/modules/intelligence/agents/chat_agents/system_agents/debug_agent.py

```
@@ -54,596 +113,45 @@ def _build_agent(self, ctx: Optional[ChatContext] = None) -> ChatAgent:
-                "update_todo_status",
-                "remove_todo",
-                "add_subtask",
-                "set_dependency",
-                "get_available_tasks",
-                "add_requirements",
-                "get_requirements",
-            ],
-            exclude_embedding_tools=exclude_embedding_tools,
+        logger.info(
+            "DebugAgent routing",
+            supports_pydantic=supports_pydantic,
```

Ruff (0.15.15)

[error] Undefined name `supports_pydantic`

(F821)